### ISSUE TITLE
Checkm test aborts with "AttributeError: Options instance has no attribute 'force_overwrite'"

### ISSUE THREAD

**Issue by jvollme** [Ecogenomics/CheckM#190]
I have just reinstalled checkm in a new conda environment using conda.
Trying to test checkm with `checkm test ~/checkm_test_results`  gives me the following error:
```
[Step 1]: Verifying tree command.

*******************************************************************************
 [CheckM - tree] Placing bins in reference genome tree.
*******************************************************************************


Unexpected error: <type 'exceptions.AttributeError'>
Traceback (most recent call last):
  File "/home/kit/ibg/ww5070/tools/miniconda2/envs/py27_env/bin/checkm", line 711, in <module>
    checkmParser.parseOptions(args)
  File "/home/kit/ibg/ww5070/tools/miniconda2/envs/py27_env/lib/python2.7/site-packages/checkm/main.py", line 1319, in parseOptions
    self.test(options)
  File "/home/kit/ibg/ww5070/tools/miniconda2/envs/py27_env/lib/python2.7/site-packages/checkm/main.py", line 1207, in test
    verifyEcoli.run(self, options.output_dir)
  File "/home/kit/ibg/ww5070/tools/miniconda2/envs/py27_env/lib/python2.7/site-packages/checkm/test/test_ecoli.py", line 66, in run
    parser.tree(options)
  File "/home/kit/ibg/ww5070/tools/miniconda2/envs/py27_env/lib/python2.7/site-packages/checkm/main.py", line 120, in tree
    if not options.force_overwrite:
AttributeError: Options instance has no attribute 'force_overwrite'
```

As far as i can see, this is different from the checkm test errors reported so far.

is this a trivial problem in the test script, or could there be something wrong with my conda environment?

BTW: I checked if this could be caused by the "checkm_test_results" directory potentially already existing: I get this result even when i double check that this directory did not already exist

**Comment by dparks1134** [comment]
It is a trivial problem. All this test does is ensure CheckM can run from start to finish (i.e., it is a test that all dependencies have been install). It is intended to help sys admins test installation of the software and is not a validation of correctness.

**Comment by jvollme** [comment]
Thanks, so this is possibly just a bug in the test-script?
Actually I really wanted to test the installation, since i had problems with my conda environments. Some packages in the cache (including checkm) appeared to be broken and wouldn't be fixed even if i tried to tell conda to ignore the cached versions or to uninstall checkm and reinstall it (i could not even uninstall checkm via conda--> got a "broken-package" Error). So I had to completely delete a whole bunch of environments, just to ensure that the packages would then be removed from the cache  with `conda clean --all`.

So now after creating new environments with conda, where the `checkm-genome` package (among others) was freshly downloaded and reinstalled, i wanted to test if everything is fine now. So this checkm error gave me some grounds for worry, that there may STILL be problems.

But if this is just a bug in the test-script, I'll just ignore it and hope for the best.

**Comment by dparks1134** [comment]
Yes, it is just a bug with the test script. A good test of the installation is to run a single genome through the "lineage_wf" command of CheckM. If this runs to completion all dependencies are properly installed.

**Comment by l-hahn** [comment]
I tried by myself; when entering '/usr/local/lib/python2.7/dist-packages/checkm/test/test_ecoli.py' (resp. your path via anacond), we can clearly see, that the option before line 66 instance does not have an attribute for force overwrite. One workaround is to add before the line 'parser.tree(options)' another line containing 'options.force_overwrite = False'. This may be forgotton?

**Comment by dparks1134** [comment]
This will be fixed in v1.0.15 which I plan to release later today.
